RecView.cpp

```diff
@@ -120,15 +116,29 @@ void RecView::OnPrint(CDC* pDC, CPrintInfo* pInfo)
 		pDC->TextOut(pageWidth / 4, yPos, rs.m_name);
 		if (rs.m_manager) {
 			manager = _T("x");
+			pDC->TextOut((pageWidth / 2), yPos, manager);
 		}
-		pDC->TextOut(2 * (pageWidth / 4), yPos, manager);
-
-		yPos += borderHeight;
-		pDC->MoveTo(leftMargin, yPos);
-		pDC->LineTo(rightMargin + 30, yPos);
 		
-		yPos += lineHeight;
 		rs.MoveNext();
+		if(!rs.IsEOF()) 
+		{
+			pDC->SelectObject(pen);

```

Comment:
`pDC->SelectObject(&pen);` - isto kao u liniji 108 (na žalost se i ovo kompajlira).
Ali ne treba vam ovaj poziv uopće, pa već ste selektirali _dashed_ pen prije petlje.

Reply:
Maknuti ću to. Kad dodajem stvari zaboravim maknuti stare. Jeli to stvara taj error?

Reply:
Da, u MFC je ovako zamišljeno (`SelectObject` sa pointerima). 
Ova verzija koja prima objekt/handle [nije uopće dokumentirana](https://learn.microsoft.com/en-us/cpp/mfc/reference/cdc-class?view=msvc-170#selectobject) (i ima drukčija očekivanja - radi, ali se okine `ASSERT` u Debug verziji).